### PR TITLE
 Add info about supported data types 

### PR DESCRIPTION
**Pull request by hariso**

### CODE REVIEW

File: docs/connectors/building-connectors/data-types.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: "Data types"
```
Comment (lovromazgon) [inline docs/connectors/building-connectors/data-types.mdx:2]: I'm thinking if this is the right place for this? Sure you care about raw vs structured data when you're developing connectors, but also when you're using conduit. WDYT about moving this to the [OpenCDC record](https://conduit.io/docs/features/opencdc-record) page?

Comment (hariso) [reply docs/connectors/building-connectors/data-types.mdx:2]: Sounds reasonable to me. :+1:

File: docs/features/opencdc-record.mdx [hunk 2]
```diff
@@ -56,9 +80,84 @@ When processing records in Conduit, you can always expect a similar structure to
...
+```
+
+## Structured data
+
+Structured data in Conduit is a map in which the keys are field names and values
+are field values. An example of that is the below record's `.Payload.After`
+field: 
+
+
+```json5
```
Comment (lovromazgon) [inline docs/features/opencdc-record.mdx:117]:
```suggestion
```json
```

File: docs/features/opencdc-record.mdx [hunk 3]
```diff
@@ -56,9 +80,84 @@ When processing records in Conduit, you can always expect a similar structure to
...
 :::note
-`.Position`, `.Key`, and `.Payload.Before` are represented as `Base64` encoded in the example above because these will be a byte slice when represented as JSON.
+`.Position`, `.Key`, and `.Payload.Before` are represented as `Base64` encoded
+in the example above because these will be a byte slice when represented as
+JSON.
 :::
 
+## Data types
+
+
```
Comment (lovromazgon) [inline docs/features/opencdc-record.mdx:90]: Redundant new line
```suggestion

```

File: docs/features/opencdc-record.mdx [hunk 4]
```diff
@@ -56,9 +80,84 @@ When processing records in Conduit, you can always expect a similar structure to
...
+opencdc.RawData([]byte{1, 3, 5})
+```
+
+## Structured data
+
+Structured data in Conduit is a map in which the keys are field names and values
+are field values. An example of that is the below record's `.Payload.After`
+field: 
+
+
```
Comment (lovromazgon) [inline docs/features/opencdc-record.mdx:116]: Redundant new line
```suggestion

```

File: docs/features/opencdc-record.mdx [hunk 5]
```diff
@@ -56,9 +80,84 @@ When processing records in Conduit, you can always expect a similar structure to
...
+* `bool`
+* `int`, `int32`, `int64`, `uint`, `uint32`, `uint64`
+* `float32`, `float64`
+* `string`
+* `[]byte` (stored as a string, base64-encoded)
+* `map[string]interface{}` (a map of strings to any of the values that are supported)
+* `[]interface{}` (a slice of any value that is supported)
+
+A notable limitation is timestamps, i.e. `time.Time` values are not supported.
```
Comment (lovromazgon) [inline docs/features/opencdc-record.mdx:159]: They _are_ supported now that we have schema support!

Comment (lovromazgon) [reply docs/features/opencdc-record.mdx:155]: I realized these listed types explain what's supported if schemas are disabled. This is fine.

File: docs/features/opencdc-record.mdx (hunk 2 above)
Comment (hariso) [reply docs/features/opencdc-record.mdx]: With `json5` there are no complaints about the comments in JSON:) (in GitHub or in the editor).

**Pull request merged**